### PR TITLE
Remove heavy metrics generation.

### PR DESCRIPTION
**Pull request by Andrei-Dolgolev**
## Changes
Remove transactions_in transactions_out value_in value_out stats generations metrics.
Save implementation in https://github.com/bugout-dev/moonstream/tree/generic-metrics-arhive

### CODE REVIEW

File: crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py [hunk 1]
```diff
@@ -802,14 +641,7 @@ def stats_generate_handler(args: argparse.Namespace):
...
 
-                        s3_data_object["generic"] = generate_metrics(
-                            db_session=db_session,
-                            blockchain_type=blockchain_type,
-                            address=address,
-                            timescale=timescale,
-                            metrics=generic_metrics_names,
-                            start=start_date,
-                        )
+                        s3_data_object["generic"] = {}
```
Comment (zomglings) [inline crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py:644]: Does this break the frontend?

Comment (Andrei-Dolgolev) [reply crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py:644]: No not breaking  dict leght must be more then 0
```javascript 
  const genericKeys = useMemo(
    () =>
      Object.keys(data?.generic ?? {}).length > 0
        ? Object.keys(data?.generic)
        : undefined,
    [data]
  );
{data?.generic && genericKeys && (
```
It mainly the case when you not select any checkboxes)

Comment (zomglings) [reply crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py:644]: Cool. Approving.

**Review by zomglings — APPROVED** [review]
lgtm

**Pull request merged**